gdcdictionary/schemas/sample.yaml

```diff
@@ -668,28 +665,18 @@ properties:
       $ref: "_terms.yaml#/tumor_descriptor"
     enum:
       - Metastatic
-      - Not Applicable
       - Primary
       - Recurrence
       - Xenograft
       - NOS
       - Unknown
+      - Not Applicable
```

Comment:
Since we're going to have a fair number of non-tumor samples, do we want this to be more explicit that it's Normal/Not Tumor? Or best way to represent that if we get rid of the combined `sample_type`?

Reply:
Would we even want to have that field populated? {"tumor_descriptor":"ITSNOTATUMAH"}

Reply:
I had missed `tissue_type` before - now I think either leave as-is or even remove `Not Applicable` as an option as it's not a required field?

Reply:
Removed `Not Applicable` from `tumor_descriptor` enum.